tools/scss/palette.scss

```diff
@@ -1,3 +1,3 @@
-@import "styles/variables_palette";
+@import "styles/variables_palette_fetch";
```

Comment:
why does microsite refer to a look and feel from a project that it does not depend on?